draft/draft-ietf-bfd-rfc9127-bis-00.xml

```diff
@@ -5,7 +5,7 @@
 <?rfc tocdepth="6"?>
 <?rfc symrefs="yes"?>
 <?rfc sortrefs="yes"?>
-<rfc category="std" docName="draft-ietf-bfd-rfc9127bis-latest" ipr="trust200902">
+<rfc category="std" docName="draft-ietf-bfd-rfc9127-bis-00" ipr="trust200902" obsoletes="9127">
```

Comment:
Should this be bis-01 (and the file too)? or maybe I misunderstand the generated files.

Reply:
You are right. It is partly automagically, and partly assigned. I fixed the assigned part, and now it should generate a bis-01 file. However, if you look at the diffs, you will see more differences than you care for.

Let me upload the rfcdiff file which would be easier to read/compare.